index.src.html

```diff
@@ -276,32 +285,58 @@ <h4 id="shortlinks">Navigation</h4>
 <section>
   <h2 id="framework">Framework</h2>
 
-  An IPv4 address is a <dfn export local-lt="private">private address</dfn> if
-  it matches the <a>private address space</a> defined in Section 3 of
-  [[!RFC1918]], a <dfn export local-lt="local">local address</dfn> if it matches
```

Comment:
I wonder if this means that we need to rename the spec... :)

Reply:
Anne filed issue #34 about that :)